### ISSUE TITLE
Encountering an exception when attempting to mine iron_ore at -54 depth

### ISSUE THREAD

**Issue by ghost** [cabaletta/baritone#4148]
## Some information
Operating system: Windows 11
Java version: 17
Minecraft version: 1.18.2
Baritone version: baritone-api-forge-1.8.5
Forge version: 40.2.10
Other mods (if used): 
```
jei-1.18.2-forge-10.2.1.1005.jar
Xaeros_Minimap_23.8.2_Forge_1.18.2.jar
XaerosWorldMap_1.34.1_Forge_1.18.2.jar
InventoryProfilesNext-forge-1.18.2-1.10.7.jar
createcafe-1.10.2-1.18.2.jar
create_jetpack-forge-1.5.2.jar
create-1.18.2-0.5.1.e.jar
Jade-1.18.2-forge-5.3.0.jar
BetterF3-3.0.0-Forge-1.18.2.jar
BBOutlineReloaded-2.6-1.18.2-forge.jar
NaturesCompass-1.18.2-1.9.7-forge.jar
I18nUpdateMod-3.5.0-all.jar
TConstruct-1.18.2-3.6.4.113.jar
tconplanner-1.18.2-1.2.0.jar
yesstevemodel-1.18.2-release-1.1.5-hotfix-2.jar
MouseTweaks-forge-mc1.18-2.21.jar
villagespawnpoint-1.18.2-4.0.jar
Mantle-1.18.2-1.9.45.jar
jecharacters-1.18.2-4.3.13.jar
carryon-1.18.2-1.17.1.12.jar
gravestone-1.18.2-1.0.2.jar
CustomSkinLoader_ForgeActive-14.17.jar
modnametooltip-1.18.1-1.18.0.jar
TravelersBackpack-1.18.2-7.1.41.jar
Quark-3.2-358.jar
appleskin-forge-mc1.18.2-2.5.1.jar
waystones-forge-1.18.2-10.2.1.jar
StorageDrawers-1.18.2-10.2.1.jar
atchouli-1.18.2-71.1.jar
ImmersiveEngineering-1.18.2-8.4.0-161.jar
MmmMmmMmmMmm-1.18.2-1.5.2.jar
rubidium-0.5.6.jar
AutoRegLib-1.7-53.jar
balm-3.2.6.jar
baritone-api-forge-1.8.5.jar
cloth-config-6.5.102-forge.jar
collective-1.18.2-6.66.jar
createaddition-1.18.2-1.0.0.jar
curios-forge-1.18.2-5.0.9.1.jar
Forgematica-0.1.7-mc1.18.2.jar
JadeAddons-1.18.2-forge-2.5.0.jar
JustEnoughResources-1.18.2-0.14.2.206.jar
kotlinforforge-3.12.0-all.jar
libIPN-forge-1.18.2-4.0.0.jar
MaFgLib-0.1.7-mc1.18.2.jar
oculus-mc1.18.2-1.6.4.jar
PRP-Arch-Forge-1.0.1-Hotfix-1.18.x.jar
sliceanddice-forge-1.3.2.jar
touhoulittlemaid-1.18.2-release-1.1.1.jar
```

## Exception, error or logs

```
[23:32:51] [Render thread/INFO] [minecraft/ChatComponent]: [CHAT] [Baritone] > mine immersiveengineering:ore_aluminum
[23:32:51] [Render thread/INFO] [minecraft/ChatComponent]: [CHAT] [Baritone] Mining [BlockOptionalMeta{block=Block{immersiveengineering:ore_aluminum},properties={}}]
java.lang.ArrayIndexOutOfBoundsException
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
	at java.base/java.util.concurrent.ForkJoinTask.getThrowableException(ForkJoinTask.java:564)
	at java.base/java.util.concurrent.ForkJoinTask.reportException(ForkJoinTask.java:591)
	at java.base/java.util.concurrent.ForkJoinTask.invoke(ForkJoinTask.java:689)
	at java.base/java.util.stream.SliceOps$1.opEvaluateParallelLazy(SliceOps.java:151)
	at java.base/java.util.stream.AbstractPipeline.sourceSpliterator(AbstractPipeline.java:434)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:233)
	at java.base/java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:682)
	at TRANSFORMER/baritoe@1.8.5/baritone.r.a(Unknown Source)
	at TRANSFORMER/baritoe@1.8.5/baritone.r.scanChunkRadius(Unknown Source)
	at TRANSFORMER/baritoe@1.8.5/baritone.eh.a(Unknown Source)
	at TRANSFORMER/baritoe@1.8.5/baritone.eh.a(Unknown Source)
	at TRANSFORMER/baritoe@1.8.5/baritone.eh.mine(Unknown Source)
	at TRANSFORMER/baritoe@1.8.5/baritone.api.process.IMineProcess.mine(Unknown Source)
	at TRANSFORMER/baritoe@1.8.5/baritone.ba.execute(Unknown Source)
	at TRANSFORMER/baritoe@1.8.5/baritone.bs.execute(Unknown Source)
	at TRANSFORMER/baritoe@1.8.5/baritone.w.a(Unknown Source)
	at TRANSFORMER/baritoe@1.8.5/baritone.w.onSendChatMessage(Unknown Source)
	at TRANSFORMER/baritoe@1.8.5/baritone.bt.a(Unknown Source)
	at java.base/java.util.concurrent.CopyOnWriteArrayList.forEach(CopyOnWriteArrayList.java:807)
	at TRANSFORMER/baritoe@1.8.5/baritone.bt.onSendChatMessage(Unknown Source)
	at TRANSFORMER/minecraft@1.18.2/net.minecraft.client.player.LocalPlayer.handler$bcm000$sendChatMessage(LocalPlayer.java)
	at TRANSFORMER/minecraft@1.18.2/net.minecraft.client.player.LocalPlayer.m_108739_(LocalPlayer.java)
	at TRANSFORMER/minecraft@1.18.2/net.minecraft.client.gui.screens.Screen.m_96612_(Screen.java:404)
	at TRANSFORMER/minecraft@1.18.2/net.minecraft.client.gui.screens.Screen.m_96615_(Screen.java:393)
	at TRANSFORMER/minecraft@1.18.2/net.minecraft.client.gui.screens.ChatScreen.m_7933_(ChatScreen.java:101)
	at TRANSFORMER/minecraft@1.18.2/net.minecraft.client.KeyboardHandler.lambda$keyPress$4(KeyboardHandler.java:363)
	at TRANSFORMER/minecraft@1.18.2/net.minecraft.client.gui.screens.Screen.m_96579_(Screen.java:528)
	at TRANSFORMER/minecraft@1.18.2/net.minecraft.client.KeyboardHandler.m_90893_(KeyboardHandler.java:353)
	at TRANSFORMER/minecraft@1.18.2/net.minecraft.client.KeyboardHandler.m_167831_(KeyboardHandler.java:452)
	at TRANSFORMER/minecraft@1.18.2/net.minecraft.util.thread.BlockableEventLoop.execute(BlockableEventLoop.java:103)
	at TRANSFORMER/minecraft@1.18.2/net.minecraft.client.KeyboardHandler.m_90938_(KeyboardHandler.java:451)
	at MC-BOOTSTRAP/org.lwjgl.glfw/org.lwjgl.glfw.GLFWKeyCallback$Container.invoke(GLFWKeyCallback.java:82)
	at TRANSFORMER/carryon@**.**.**.**/tschipp.carryon.common.helper.KeyboardCallbackWrapper.keyCallback(KeyboardCallbackWrapper.java:28)
	at MC-BOOTSTRAP/org.lwjgl.glfw/org.lwjgl.glfw.GLFWKeyCallbackI.callback(GLFWKeyCallbackI.java:37)
	at MC-BOOTSTRAP/org.lwjgl/org.lwjgl.system.JNI.invokeV(Native Method)
	at MC-BOOTSTRAP/org.lwjgl.glfw/org.lwjgl.glfw.GLFW.glfwPollEvents(GLFW.java:3101)
	at TRANSFORMER/minecraft@1.18.2/com.mojang.blaze3d.systems.RenderSystem.m_69495_(RenderSystem.java:197)
	at TRANSFORMER/minecraft@1.18.2/com.mojang.blaze3d.platform.Window.m_85435_(Window.java:333)
	at TRANSFORMER/minecraft@1.18.2/net.minecraft.client.Minecraft.m_91383_(Minecraft.java:1068)
	at TRANSFORMER/minecraft@1.18.2/net.minecraft.client.Minecraft.m_91374_(Minecraft.java:665)
	at TRANSFORMER/minecraft@1.18.2/net.minecraft.client.main.Main.main(Main.java:205)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at MC-BOOTSTRAP/fmlloader@1.18.2-40.2.10/net.minecraftforge.fml.loading.targets.CommonClientLaunchHandler.lambda$launchService$0(CommonClientLaunchHandler.java:31)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@9.1.3/cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:37)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@9.1.3/cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:53)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@9.1.3/cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:71)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@9.1.3/cpw.mods.modlauncher.Launcher.run(Launcher.java:106)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@9.1.3/cpw.mods.modlauncher.Launcher.main(Launcher.java:77)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@9.1.3/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@9.1.3/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23)
	at cpw.mods.bootstraplauncher@1.0.0/cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:149)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at io.github.zekerzhayard.forgewrapper.installer.Main.main(Main.java:58)
	at org.prismlauncher.launcher.impl.StandardLauncher.launch(StandardLauncher.java:88)
	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:126)
	at org.prismlauncher.EntryPoint.main(EntryPoint.java:71)
Caused by: java.lang.ArrayIndexOutOfBoundsException: Index -4 out of bounds for length 24
	at TRANSFORMER/baritoe@1.8.5/baritone.r.a(Unknown Source)
	at TRANSFORMER/baritoe@1.8.5/baritone.r.a(Unknown Source)
	at TRANSFORMER/baritoe@1.8.5/baritone.r.b(Unknown Source)
	at java.base/java.util.stream.ReferencePipeline$7$1.accept(ReferencePipeline.java:273)
	at java.base/java.util.ArrayList$ArrayListSpliterator.tryAdvance(ArrayList.java:1602)
	at java.base/java.util.stream.ReferencePipeline.forEachWithCancel(ReferencePipeline.java:129)
	at java.base/java.util.stream.AbstractPipeline.copyIntoWithCancel(AbstractPipeline.java:527)
	at java.base/java.util.stream.SliceOps$SliceTask.doLeaf(SliceOps.java:629)
	at java.base/java.util.stream.SliceOps$SliceTask.doLeaf(SliceOps.java:572)
	at java.base/java.util.stream.AbstractShortCircuitTask.compute(AbstractShortCircuitTask.java:115)
	at java.base/java.util.concurrent.CountedCompleter.exec(CountedCompleter.java:754)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)
baritone.api.command.exception.CommandUnhandledException: java.lang.ArrayIndexOutOfBoundsException
	at TRANSFORMER/baritoe@1.8.5/baritone.bs.execute(Unknown Source)
	at TRANSFORMER/baritoe@1.8.5/baritone.w.a(Unknown Source)
	at TRANSFORMER/baritoe@1.8.5/baritone.w.onSendChatMessage(Unknown Source)
	at TRANSFORMER/baritoe@1.8.5/baritone.bt.a(Unknown Source)
	at java.base/java.util.concurrent.CopyOnWriteArrayList.forEach(CopyOnWriteArrayList.java:807)
	at TRANSFORMER/baritoe@1.8.5/baritone.bt.onSendChatMessage(Unknown Source)
	at TRANSFORMER/minecraft@1.18.2/net.minecraft.client.player.LocalPlayer.handler$bcm000$sendChatMessage(LocalPlayer.java)
	at TRANSFORMER/minecraft@1.18.2/net.minecraft.client.player.LocalPlayer.m_108739_(LocalPlayer.java)
	at TRANSFORMER/minecraft@1.18.2/net.minecraft.client.gui.screens.Screen.m_96612_(Screen.java:404)
	at TRANSFORMER/minecraft@1.18.2/net.minecraft.client.gui.screens.Screen.m_96615_(Screen.java:393)
	at TRANSFORMER/minecraft@1.18.2/net.minecraft.client.gui.screens.ChatScreen.m_7933_(ChatScreen.java:101)
	at TRANSFORMER/minecraft@1.18.2/net.minecraft.client.KeyboardHandler.lambda$keyPress$4(KeyboardHandler.java:363)
	at TRANSFORMER/minecraft@1.18.2/net.minecraft.client.gui.screens.Screen.m_96579_(Screen.java:528)
	at TRANSFORMER/minecraft@1.18.2/net.minecraft.client.KeyboardHandler.m_90893_(KeyboardHandler.java:353)
	at TRANSFORMER/minecraft@1.18.2/net.minecraft.client.KeyboardHandler.m_167831_(KeyboardHandler.java:452)
	at TRANSFORMER/minecraft@1.18.2/net.minecraft.util.thread.BlockableEventLoop.execute(BlockableEventLoop.java:103)
	at TRANSFORMER/minecraft@1.18.2/net.minecraft.client.KeyboardHandler.m_90938_(KeyboardHandler.java:451)
	at MC-BOOTSTRAP/org.lwjgl.glfw/org.lwjgl.glfw.GLFWKeyCallback$Container.invoke(GLFWKeyCallback.java:82)
	at TRANSFORMER/carryon@**.**.**.**/tschipp.carryon.common.helper.KeyboardCallbackWrapper.keyCallback(KeyboardCallbackWrapper.java:28)
	at MC-BOOTSTRAP/org.lwjgl.glfw/org.lwjgl.glfw.GLFWKeyCallbackI.callback(GLFWKeyCallbackI.java:37)
	at MC-BOOTSTRAP/org.lwjgl/org.lwjgl.system.JNI.invokeV(Native Method)
	at MC-BOOTSTRAP/org.lwjgl.glfw/org.lwjgl.glfw.GLFW.glfwPollEvents(GLFW.java:3101)
	at TRANSFORMER/minecraft@1.18.2/com.mojang.blaze3d.systems.RenderSystem.m_69495_(RenderSystem.java:197)
	at TRANSFORMER/minecraft@1.18.2/com.mojang.blaze3d.platform.Window.m_85435_(Window.java:333)
	at TRANSFORMER/minecraft@1.18.2/net.minecraft.client.Minecraft.m_91383_(Minecraft.java:1068)
	at TRANSFORMER/minecraft@1.18.2/net.minecraft.client.Minecraft.m_91374_(Minecraft.java:665)
	at TRANSFORMER/minecraft@1.18.2/net.minecraft.client.main.Main.main(Main.java:205)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at MC-BOOTSTRAP/fmlloader@1.18.2-40.2.10/net.minecraftforge.fml.loading.targets.CommonClientLaunchHandler.lambda$launchService$0(CommonClientLaunchHandler.java:31)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@9.1.3/cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:37)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@9.1.3/cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:53)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@9.1.3/cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:71)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@9.1.3/cpw.mods.modlauncher.Launcher.run(Launcher.java:106)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@9.1.3/cpw.mods.modlauncher.Launcher.main(Launcher.java:77)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@9.1.3/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@9.1.3/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23)
	at cpw.mods.bootstraplauncher@1.0.0/cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:149)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at io.github.zekerzhayard.forgewrapper.installer.Main.main(Main.java:58)
	at org.prismlauncher.launcher.impl.StandardLauncher.launch(StandardLauncher.java:88)
	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:126)
	at org.prismlauncher.EntryPoint.main(EntryPoint.java:71)
Caused by: java.lang.ArrayIndexOutOfBoundsException
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
	at java.base/java.util.concurrent.ForkJoinTask.getThrowableException(ForkJoinTask.java:564)
	at java.base/java.util.concurrent.ForkJoinTask.reportException(ForkJoinTask.java:591)
	at java.base/java.util.concurrent.ForkJoinTask.invoke(ForkJoinTask.java:689)
	at java.base/java.util.stream.SliceOps$1.opEvaluateParallelLazy(SliceOps.java:151)
	at java.base/java.util.stream.AbstractPipeline.sourceSpliterator(AbstractPipeline.java:434)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:233)
	at java.base/java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:682)
	at TRANSFORMER/baritoe@1.8.5/baritone.r.a(Unknown Source)
	at TRANSFORMER/baritoe@1.8.5/baritone.r.scanChunkRadius(Unknown Source)
	at TRANSFORMER/baritoe@1.8.5/baritone.eh.a(Unknown Source)
	at TRANSFORMER/baritoe@1.8.5/baritone.eh.a(Unknown Source)
	at TRANSFORMER/baritoe@1.8.5/baritone.eh.mine(Unknown Source)
	at TRANSFORMER/baritoe@1.8.5/baritone.api.process.IMineProcess.mine(Unknown Source)
	at TRANSFORMER/baritoe@1.8.5/baritone.ba.execute(Unknown Source)
	... 48 more
Caused by: java.lang.ArrayIndexOutOfBoundsException: Index -4 out of bounds for length 24
	at TRANSFORMER/baritoe@1.8.5/baritone.r.a(Unknown Source)
	at TRANSFORMER/baritoe@1.8.5/baritone.r.a(Unknown Source)
	at TRANSFORMER/baritoe@1.8.5/baritone.r.b(Unknown Source)
	at java.base/java.util.stream.ReferencePipeline$7$1.accept(ReferencePipeline.java:273)
	at java.base/java.util.ArrayList$ArrayListSpliterator.tryAdvance(ArrayList.java:1602)
	at java.base/java.util.stream.ReferencePipeline.forEachWithCancel(ReferencePipeline.java:129)
	at java.base/java.util.stream.AbstractPipeline.copyIntoWithCancel(AbstractPipeline.java:527)
	at java.base/java.util.stream.SliceOps$SliceTask.doLeaf(SliceOps.java:629)
	at java.base/java.util.stream.SliceOps$SliceTask.doLeaf(SliceOps.java:572)
	at java.base/java.util.stream.AbstractShortCircuitTask.compute(AbstractShortCircuitTask.java:115)
	at java.base/java.util.concurrent.CountedCompleter.exec(CountedCompleter.java:754)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)
[23:32:51] [Render thread/INFO] [minecraft/ChatComponent]: [CHAT] [Baritone] An unhandled exception occurred. The error is in your game's log, please report this at https://github.com/cabaletta/baritone/issues
Log upload triggered at: 07 Oct 2023 23:33:00  0900
```

## How to reproduce

I tested it, and if you try to mine ores that do not generate deep underground, this error occurs. Although this won't have a significant impact, after the error occurs, the automatic pathfinding will attempt to move to a peculiar location.

## Modified settings

None (I reseted all setting to default)

## Final checklist
- [x] I know how to properly use check boxes 
- [x] I have included the version of Minecraft I'm running, baritone's version and forge mods (if used).
- [x] I have included logs, exceptions and / or steps to reproduce the issue.
- [x] I have not used any OwO's or UwU's in this issue.

**Comment by Nekiplay** [comment]
Fixed in #4145

**Comment by ZacSharp** [comment]
@Nekiplay They are on 1.18.2 so that won't help them.

@NagaseYami if you want to fix this for yourself you'll need to build Baritone yourself from the 1.18.2 branch after applying the change from #4145.

**Comment by Nekiplay** [comment]
> @Nekiplay They are on 1.18.2 so that won't help them.
> 
> @NagaseYami if you want to fix this for yourself you'll need to build Baritone yourself from the 1.18.2 branch after applying the change from #4145.

I know there is no fix in the 1.18.2 branch, I myself use a modified version of 1.20.1 with the fix

**Comment by ZacSharp** [comment]
> I know there is no fix in the 1.18.2 branch, I myself use a modified version of 1.20.1 with the fix

The fix also probably will not be ported to 1.18.2 at all so using a modified version is the only option to get it on 1.18.2; waiting for the next release won't work. Otherwise I would be completely fine closing this as fixed right away (and I will close it if no further questions arise).
